webapp/components/post_view/components/reaction.jsx

```diff
@@ -57,7 +65,32 @@ export default class Reaction extends React.Component {
         }
 
         let tooltip;
-        if (users.length > 1) {
+        if (otherUsers > 0) {
+            if (users.length > 0) {
+                tooltip = (
+                    <FormattedHTMLMessage
```

Comment:
I was trying to avoid changing the existing localization strings, but I could definitely do that seeing as I'm already adding two new ones.